Web/starter_project_barbies/pages/team/Team.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import TeamsImage from '../../public/teams_page_img.svg'
+import Image from 'next/image'
+import TeamMemberCard from "@/components/team/TeamMemberCard";
```

Comment:
this does not exist yet
so you have to make sure this does not merge before the other one does

Reply:
The proffessional way to handle this would be to create a PR that defines the common types and components that both PR's use and merge than first so that the other PR's can work independently as much as possible

Reply:
> this does not exist yet so you have to make sure this does not merge before the other one does

Since I changed the base branch for this PR, it merges to the one containing the component.

Reply:
> The proffessional way to handle this would be to create a PR that defines the common types and components that both PR's use and merge than first so that the other PR's can work independently as much as possible

The other branch was created to create the common component and this branch merges to the one containing the common component, if it made sense.